SU2_CFD/src/solvers/CTurbSASolver.cpp

```diff
@@ -887,7 +896,10 @@ void CTurbSASolver::BC_Engine_Exhaust(CGeometry *geometry, CSolver **solver_cont
 
 //      /*--- Viscous contribution, commented out because serious convergence problems ---*/
 //
-//      visc_numerics->SetCoord(geometry->nodes->GetCoord(iPoint), geometry->nodes->GetCoord(iPoint));
```

Comment:
Here again.
`Point_Normal` is not defined by the surrounding code, but this also applies to other commented-out sections.